sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -426,6 +433,7 @@ export interface ServiceBusReceivedMessage extends ServiceBusMessage {
     readonly lockToken?: string;
     readonly _rawAmqpMessage: AmqpAnnotatedMessage;
     readonly sequenceNumber?: Long_2;
+    readonly state?: ServiceBusMessageState;
```

Comment:
Why enum?

Can instead do ` "active" | "deferred" | "scheduled"`

Reply:
with enum I can cast the values (0,1,2) returned from the service.

Reply:
Good point on using the user-friendly literal union! It is more consistent with the public api surface.  Will update